spec/javascripts/components/projects/side_bar/side_bar_spec.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import { SideBar } from 'components/projects/SideBar/index';
+import SideBar from 'components/projects/SideBar/index';
```

Comment:
You don't need to put the `index` in the path. Here's good content about require paths https://nodejs.org/api/modules.html#modules_all_together